src/storage/resourceLoaderInitialization.ts

```diff
@@ -0,0 +1,24 @@
+import { Disposable } from "vscode";
+import { CCLOUD_CONNECTION_ID, LOCAL_CONNECTION_ID } from "../constants";
+import { CCloudResourceLoader } from "./ccloudResourceLoader";
+import { LocalResourceLoader } from "./localResourceLoader";
+import { ResourceLoader } from "./resourceLoader";
+
+/* This is used by both extension initialization and also the global test suite setup.
+
```

Comment:
@alexeyraspopov Any idea why, when this function is defined in extension.ts, that it blows up the test suite a few seconds after when trying to run?